returnn/torch/engine.py

```diff
@@ -1444,6 +1466,7 @@ def _print_process(
     start_elapsed: Optional[float] = None,
     seq_idx: Optional[int] = None,
     num_seqs: Optional[int] = None,
+    complete_frac: Optional[float] = None,
```

Comment:
Do we need `num_seqs` at all when we have `complete_frac`?

Reply:
Right. I think this logic applies to some more places as well.